actioncable/lib/assets/javascripts/action_cable/connection.coffee

```diff
@@ -56,14 +56,17 @@ class ActionCable.Connection
   events:
     message: (event) ->
       {identifier, message, type} = JSON.parse(event.data)
+      ping = identifier == ActionCable.INTERNAL.identifiers.ping
```

Comment:
Thought we'd be using this when dispatching `"connected"` as well. Let's inline the variable in the `else` block like this:

```coffeescript
else
  @consumer.subscriptions.notify(identifier, "received", message)

  if identifier isnt ActionCable.INTERNAL.identifiers.ping
    @dispatchEvent("received", message, JSON.parse(identifier).channel)
```